src/query/service/src/interpreters/interpreter_insert_v2.rs

```diff
@@ -425,6 +486,39 @@ impl ValueSource {
 
         Ok(())
     }
+
+    /// Parse single row values.
+    /// If the value is an expression, an Error will occur.
+    fn parse_next_row_values<R: AsRef<[u8]>>(
+        &self,
+        field_decoder: &FieldDecoderValues,
+        reader: &mut Cursor<R>,
+        col_size: usize,
+        desers: &mut [TypeDeserializerImpl],
+    ) -> Result<()> {
+        let _ = reader.ignore_white_spaces();
+
+        // Start of the row --- '('
+        if !reader.ignore_byte(b'(') {
+            return Err(ErrorCode::BadDataValueType(
+                "Must start with parentheses".to_string(),
+            ));
+        }
+        for col_idx in 0..col_size {
+            let _ = reader.ignore_white_spaces();
+            let deser = desers
+                .get_mut(col_idx)
+                .ok_or_else(|| ErrorCode::BadBytes("Deserializer is None"))?;
```

Comment:
```suggestion
                .ok_or_else(|| ErrorCode::Internal("Deserializer is None"))?;
```

Reply:
/// Internal means this is the internal error that no action
    /// can be taken by neither developers or users.
    /// In most of the time, they are code bugs.